frontend/docker/.env.example

```
@@ -0,0 +1,26 @@
+# Copy this file to frontend/docker/.env before running frontend Docker commands.
+
+COMPOSE_PROJECT_NAME=nijigen-video-site-frontend
+
+WEB_DUMMY_IMAGE=nginx:alpine
+FRONTEND_HTTP_PORT=5173
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FRONTEND_HTTP_PORT key should go before the WEB_DUMMY_IMAGE key

(UnorderedKey)